doc/read-the-docs-site/background/index.rst

```diff
@@ -1,5 +1,11 @@
-Background Information
-======================
+Foundational credential concepts
```

Comment:
I believe we need more details for the introduction page.

First, we need to define the credential management system. What is it? Who is it for? Why should one use it?

Then, explain the concepts of keys and continue with the short comparison.

Reply:
I think these questions are at least partially addressed on the top-level landing page of the whole documentation system (https://credential-manager.readthedocs.io/en/latest/index.html). However, I think it could use further development to address those questions more fully.

Reply:
I'm exploring expanding on this topic. It will be in my next round of commits.